src/Pim/Bundle/EnrichBundle/Normalizer/CompletenessCollectionNormalizer.php

```diff
@@ -27,35 +29,83 @@ public function __construct(NormalizerInterface $normalizer)
 
     /**
      * {@inheritdoc}
+     *
+     * Normalized completeness collection that is returned looks like:
+     *
+     * [
+     *     [
+     *         'locale'   => 'de_DE',
+     *         'stats'    => [
+     *             'total'    => 3,
+     *             'complete' => 0,
+     *         ],
+     *         'channels' => [
+     *             'ecommerce' => [
+     *                 'completeness' => [
+     *                     'required' => 4,
+     *                     'missing' => 2,
+     *                     'ratio' => 50,
+     *                     'locale' => 'de_DE',
+     *                     'channel_code' => 'ecommerce',
+     *                     'channel_labels' => [
+     *                         'en_US' => 'Ecommerce',
+     *                         'de_DE' => 'Ecommerce',
+     *                         'fr_FR' => 'Ecommerce',
+     *                     ],
+     *                     'missing' => [
```

Comment:
Definitely something wrong here: 'missing' key collision. I'd name it missing_attributes